packages/components/src/slot-fill/stories/index.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Slot, Fill, Provider as SlotFillProvider } from '../';
+
+const meta: ComponentMeta< typeof Slot > = {
+	component: Slot,
+	title: 'Components/SlotFill',
+	subcomponents: { Fill, SlotFillProvider },
+	argTypes: {
+		name: { control: { type: null } },
+		as: { control: { type: 'text' } },
+		fillProps: { control: { type: null } },
```

Comment:
Can we get a bit more specific and actually parameterize the examples that pass data to `fillProps`? For example `age` could be its own control, regardless of the fact that it passes to `fillProps`.